routes.go

```diff
@@ -54,6 +54,8 @@ func DisplayRoutes(w http.ResponseWriter, r *http.Request) {
 		return strings.Compare(a, b)
 	})
 
+	// We still report it as a 404
+	w.WriteHeader(http.StatusNotFound)
```

Comment:
Okay but we're still not going to get the metrics coming through, shall we roll the two changes into one PR?

Reply:
No, it does go through our Prometheus middleware and gets recognized (at least on my machine it works)